src/Training/Bundle/UserNamingBundle/Api/Processor/AddUserNamingExampleForGetList.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Training\Bundle\UserNamingBundle\Api\Processor;
+
+use Oro\Component\ChainProcessor\ContextInterface;
+use Oro\Component\ChainProcessor\ProcessorInterface;
+use Training\Bundle\UserNamingBundle\Formatter\UserNameFormatter;
+
+class AddUserNamingExampleForGetList implements ProcessorInterface
+{
+    private const META_PROPERTY_KEY = 'nameExample';
+
+    private UserNameFormatter $nameFormatter;
+    public function __construct(UserNameFormatter $nameFormatter)
+    {
+        $this->nameFormatter = $nameFormatter;
+    }
+
+    public function process(ContextInterface $context)
+    {
+        if (! $context->getMetadata()?->hasProperty(self::META_PROPERTY_KEY)) {
+            return;
+        }
+
+        $result = $context->getResult();
+
+        if (! $result || !is_array($result)) {
```

Comment:
```suggestion
        if (!is_array($result)) {
```
The first check is redundant